MeshPlugin/TypeStructs.cs

```diff
@@ -0,0 +1,842 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace AssetsTools
+{
+    [StructLayout(LayoutKind.Sequential, Pack = 4)]
+    public struct Vector3 : IEquatable<Vector3>
```

Comment:
Put these in separate files as well (in their own directory). Even if it was originally like that in the AssetStudio repo, I'd rather them be separate.